Package-Config.roc

```diff
@@ -11,10 +11,14 @@ initForHost = init
 # TODO: once `roc glue` knows how to resolve them, these should move back
 # into Rbt.roc so we can stop copying the definitions over every time we make
 # a change!
-Tool : [SystemTool { name : Str }]
+SystemToolPayload : { name : Str }
+Tool : [SystemTool SystemToolPayload]
 
-Command : [Command { tool : Tool, args : List Str }]
+CommandPayload : { tool : Tool, args : List Str }
+Command : [Command CommandPayload]
 
-Job : [Job { command : Command, inputFiles : List Str, outputs : List Str }]
+JobPayload : { command : Command, inputFiles : List Str, outputs : List Str }
+Job : [Job JobPayload]
 
-Rbt : [Rbt { default : Job }]
+RbtPayload : { default : Job }
+Rbt : [Rbt RbtPayload]
```

Comment:
Why is the union needed? Could it be:
`Rbt : { default : Job }`

Or maybe an opaque type if that is wanted:
`Rbt := { default : Job }`

Same with other unions that are just of one variant.

Reply:
oh man, I would love for this not to be a union / for it to be opaque. Problem is, right now glue can't see through module boundaries, so if we make this opaque then the constructors that live in `Rbt.roc` won't be able to construct values.

I've issued https://github.com/roc-lang/rbt/issues/44 to track this so it's not in my head only.

Reply:
oh, and: it's a union because it used to be a private union. I suppose that could change, even if we don't make it opaque right now.